app/components/provider_interface/application_choice_header_component.html.erb

```diff
@@ -16,10 +16,12 @@
             <% end %>
           </h2>
           <p class="govuk-body">
-            <% if time_is_today_or_tomorrow?(application_choice.reject_by_default_at) -%>
-              You have until <%= time_today_or_tomorrow(application_choice.reject_by_default_at) %> to respond to this application. Otherwise it will be automatically rejected.
+            <% if application_choice.reject_by_default_at.today? -%>
+              This application will be automatically rejected at the end of today (<%= time_today_or_tomorrow(application_choice.reject_by_default_at) %>) if you do not make a decision.
+            <% elsif time_is_tomorrow?(application_choice.reject_by_default_at) -%>
+              This application will be automatically rejected at the end of tomorrow (<%= time_today_or_tomorrow(application_choice.reject_by_default_at) %>) if you do not make a decision.
```

Comment:
Can we extract the conditions in a method which decides if we should print `today` or `tomorrow`, as the rest of the text is the same?

```suggestion
            <% if application_choice.reject_by_default_at.today? ||  time_is_tomorrow?(application_choice.reject_by_default_at) -%>
              This application will be automatically rejected at the end of <%= some_method_that_returns_today_or_tomorrow %> (<%= time_today_or_tomorrow(application_choice.reject_by_default_at) %>) if you do not make a decision.
```

Reply:
👍